### DISCUSSION TITLE
How to plot sales per country on map scatter visualization

### DISCUSSION THREAD

**Discussion by aszenz** [perspective-dev/perspective#2569] [Q&A]
I have a dataset with iso country codes and wish to visualize it using map scatter but currently map scatter only accepts lat/long columns, is their any way to plot values per country instead ?

**Comment by sinistersnare** [comment]
One method would be to make two custom columns that map ISO country codes to lat/lon columns.

**Reply by aszenz** [reply]
Hmm, seems doable, I guess it has to be done manually for each country, can't use anything for calculation right?

**Reply by mhkeller** [reply]
You could download a world shape file, load it into a program like QGIS and have it compute the centroid of each country.
